geometry_msgs/PoseStamped target
uint32 target_id
---
bool success